ompi/dpm/dpm.c

```diff
@@ -1058,6 +1060,13 @@ int ompi_dpm_spawn(int count, const char *array_of_commands[],
             }
 #endif
 
+            /* check for 'mpi_initial_errhandler' */
+            ompi_info_get (array_of_info[i], "mpi_initial_errhandler", sizeof(init_errh) - 1, init_errh, &flag);
```

Comment:
We should revise this MPI_Info key name - if you look at the rest of them, you'll see they are all PMIx attributes. It will be somewhat confusing if we break that rule by introducing a lone exception. If we don't have an existing attribute we can use, then we should define one.

Reply:
This could be a problem:, these keyS (wdir, host, SOFT, etc)  are MPI standard forms (just like the single dash arguments). MPI standard chapter 10.3.4. 

why has the PMIX prefix been added to these in the first place?

Reply:
Largely because I was unaware that MPI had "standardized" some info keys. Feel free to add the standard forms, but keep the PMIx ones as well - makes it simpler for people who are using those for other infos.

Reply:
pr #7738